src/pages/settings/Report/RoomNamePage.js

```diff
@@ -67,26 +77,28 @@ function RoomNamePage(props) {
 
     return (
         <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-            <HeaderWithBackButton
-                title={translate('newRoomPage.roomName')}
-                onBackButtonPress={() => Navigation.goBack(ROUTES.getReportSettingsRoute(report.reportID))}
-            />
-            <Form
-                style={[styles.flexGrow1, styles.ph5]}
-                formID={ONYXKEYS.FORMS.ROOM_NAME_FORM}
-                onSubmit={(values) => Report.updatePolicyRoomNameAndNavigate(report, values.roomName)}
-                validate={validate}
-                submitButtonText={translate('common.save')}
-                enabledWhenOffline
-            >
-                <View style={styles.mb4}>
-                    <RoomNameInput
-                        inputID="roomName"
-                        autoFocus
-                        defaultValue={report.reportName}
-                    />
-                </View>
-            </Form>
+            <FullPageNotFoundView shouldShow={!Policy.isPolicyOwner(policy) && policy.role !== CONST.POLICY.ROLE.ADMIN}>
```

Comment:
@hungvu193 This is duplicating the room rename permission logic. We should create a util function to have the logic of room rename permission check and use it in both these places.